src/extractEnglishPages.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by Anshul on 3/6/2017.
+ */
+
+const path = require('path');
+const fs = require('fs');
+var franc = require('franc');
+const WARCStream = require('warc');
+var writeStream = fs.createWriteStream('./data/english.wet', { flags : 'w' });
```

Comment:
I'll make the changes. This was just for testing franc though, I intended to do these later. I just wanted to say that franc seems to be reasonably ok.